test/CMakeLists.txt

```diff
@@ -43,44 +43,52 @@ add_halide_library(taylor FROM taylor.generator)
 
 add_executable(arange_test arange_test.cpp)
 target_link_libraries(arange_test PRIVATE Halide::Halide
-                                          arange)
+                                          arange
+                                          ${MPI_CXX_LIBRARIES})
```

Comment:
Future: Transitive dependencies should be picked up from `Halide::Halide`, not specified here.